packages/xo-web/src/common/xo/index.js

```diff
@@ -2840,6 +2840,17 @@ export const reclaimSrSpace = async sr => {
     if (err?.data?.message?.includes('Operation not supported')) {
       throw new Error('Space reclaim not supported. Only supported on block based/LVM based SRs.')
     }
+    if (forbiddenOperation.is(err)) {
```

Comment:
In case we receive a `forbiddenOperation` error for another reason, we will display this modal.
Use `backupIsRunning` function to ensure we are on the case of a `backupGuard` error
`if(backupIsRunning(err, sr.$pool)`